book/04-git-server/sections/protocols.asc

```diff
@@ -46,28 +46,27 @@ Lalu, Anda dapat melakukan _push_ dan _pull_ dari dan ke repositori _remote_ sep
 Repositori berbasis _file_ ini didukung karena terlihat lebih sedarhana dan menggunakan hak akses berkas dan akses jaringan yang ada.
```

Comment:
Typo "... lebih "**sedarhana**"

Reply:
Thanks for correcting the misspelling, will change it in the next PR.